### PR TITLE
add stable + beta + canary embers in ember-try

### PR DESCRIPTION
**Pull request by stefanpenner**
- [ ] fix the few remaining failing tests

### CODE REVIEW

**Comment by rwjblue** [comment]
@stefanpenner / @lukemelia - These were already included in:

* https://github.com/emberjs/ember-collection/blob/master/config/ember-try/allowed-failures.js
* https://github.com/emberjs/ember-collection/blob/master/config/ember-try/channels.js

This is so that we can take advantage of the various builds that circle-ci gives us, see here:

https://github.com/emberjs/ember-collection/blob/master/circle.yml#L17-L20



tldr; I don't think we need this PR...

**Comment by kiwiupover** [comment]
@stefanpenner would love to see this released. Looks like the tests passed.

**Comment by stefanpenner** [comment]
@kiwiupover ya we gotta release this. If pinged this weekend, I can forsure do that.

**Comment by stefanpenner** [comment]
@rwjblue thanks for keeping me in check.

**Comment by rwjblue** [comment]
released https://github.com/emberjs/ember-collection/releases/tag/v1.0.0-alpha.7
